src/pybind/mgr/orchestrator.py

```diff
@@ -768,9 +768,9 @@ def pretty_print(self, only_header=False):
             return row_format.format("Device Path", "Type", "Size", "Rotates",
                                      "Available", "Model")
         else:
-            return row_format.format(self.id, self.type, format_bytes(self.size, 5, colored=False),
```

Comment:
there is a dedicated PR for this: #26357

Reply:
@sebastian-philipp removed this commit.